src/status_im/contexts/wallet/wallet_connect/sign_message/view.cljs

```diff
@@ -0,0 +1,79 @@
+(ns status-im.contexts.wallet.wallet-connect.sign-message.view
+  (:require [quo.core :as quo]
+            [quo.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im.common.raw-data-block.view :as raw-data-block]
+            [status-im.common.standard-authentication.core :as standard-authentication]
+            [status-im.contexts.wallet.wallet-connect.sign-message.style :as style]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]))
+
+(defn- close-sheet
+  []
+  (rf/dispatch [:navigate-back]))
+
+(defn- on-auth-success
+  [password]
+  (rf/dispatch [:hide-bottom-sheet])
+  (rf/dispatch [:wallet-connect/respond-current-session password]))
+
+;; NOTE: this will be a reusable component
+(defn- header
+  [{:keys [emoji customization-color name]}]
+  [rn/view
+   {:style {:padding-vertical 12}}
+   [quo/text
+    {:size   :heading-1
+     :weight :semi-bold}
+    [rn/view {:style {:margin-top -4}}
+     [quo/summary-tag
+      {:label "dapp placeholder"}]]
+    (i18n/label :t/wallet-connect-sign-header)
+    [rn/view {:style {:padding-top 4}}
+     [quo/summary-tag
+      {:type                :account
+       :emoji               emoji
+       :label               name
+       :customization-color customization-color}]]]])
+
+(defn view
+  []
+  (let [bottom                                   (safe-area/get-bottom)
+        address                                  (rf/sub [:wallet-connect/current-request-address])
+        {:keys [name emoji customization-color]} (rf/sub [:wallet-connect/account-details-by-address
+                                                          address])
+        display-data                             (rf/sub [:wallet-connect/current-request-display-data])]
```

Comment:
A minor suggestion for performance, `display-data` is only used by `raw-data-block/view`. You could create a separate component wrapping `raw-data-block/view` and subscribe from there, thus skipping processing the entire `view` again.